web/src/integTest/java/com/microsoft/applicationinsights/azuretest/CompareTelemetryWithAzureExportTests.java

```diff
@@ -0,0 +1,342 @@
+package com.microsoft.applicationinsights.azuretest;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import com.microsoft.azure.storage.queue.CloudQueue;
+import com.microsoft.azure.storage.queue.CloudQueueClient;
+import com.microsoft.azure.storage.queue.CloudQueueMessage;
+import org.apache.commons.lang3.time.StopWatch;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import javax.xml.bind.JAXBException;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Created by moralt on 30/4/2015.
+ */
+public class CompareTelemetryWithAzureExportTests {
+    String runId = UUID.randomUUID().toString();
+    String configEnvironmentVariable = "INTEGRATION_TEST_CONFIGURATION";
+    IntegrationTestConfiguration config;
+    private static final int millisecondsInSecond = 1000;
+
+    /**
+     * Sends GET requests to server and expects to get similar result from app insights
+     * @throws Exception
+     */
+    @Test
+    public void sendHttpRequestTest() throws Exception {
+
+        initConfiguration();
+        clearQueue();
+
+        ArrayList<TelemetryItem> expectedTelemetries = sentHttpGetRequests();
+        ArrayList<TelemetryItem> realTelemetries = getTelemetryFromAzure(DocumentType.Requests);
+
+        ArrayList<TelemetryItem> missingTelemetry = getMissingExpectedTelemetry(realTelemetries, expectedTelemetries);
+        if (missingTelemetry.size() > 0) {
+            for (TelemetryItem item : missingTelemetry) {
+                System.out.println("Didn't find matching item in real telemetry for request of URI " + item.getProperty("uri"));
+            }
+
+            throw new Exception("Didn't find match for " + missingTelemetry.size() + " items");
+        }
+        else {
+            System.out.println("Test passed successfully");
+        }
+    }
+
+    private void initConfiguration() throws JAXBException {
+        String location = System.getenv(configEnvironmentVariable);
+        Map<String, String> env =  System.getenv();
+        System.out.println("Configuration file located in " + location);
+
+        try {
+           config = IntegrationTestConfiguration.load(location);
+        }
+        catch(JAXBException e) {
+            System.out.println("Failed to load configuration file " + location);
+            throw e;
+        }
+    }
+
+    /**
+     * Clears the Azure queue from all existing messages
+     * @throws Exception
+     */
+    private void clearQueue() throws Exception {
```

Comment:
method name does not reflect which Queue is to be clear. Please rename to clearAzureQueue.

Also, this function assumes that `config` had been initialized. However, it would be better to make this function more generic by having it accept the relevant parameters, i.e. `account`, `queueClient` and `queueName`.